packages/playwright/src/generators/configuration/files/playwright.config.ts.template

```diff
@@ -1,7 +1,6 @@
 import { defineConfig, devices } from '@playwright/test';
 import { nxE2EPreset } from '@nx/playwright/preset';
-<% if(!webServerCommand || !webServerAddress) { %>// eslint-disable-next-line @typescript-eslint/no-unused-vars <% } %>
-import { workspaceRoot } from '@nx/devkit';
+<% if (webServerCommand && webServerAddress) { %>import { workspaceRoot } from '@nx/devkit';<% } %>
```

Comment:
This import is unused unless both options are set, which leads to typecheck errors in the new setup. We should just not import it if we're not going to use it.